### PR TITLE
add requirements to the setup.py

### PR DESCRIPTION
**Pull request by itziakos**

### CODE REVIEW

**Comment by itziakos** [comment]
@dpinte can I get a quick review

File: setup.py [hunk 1]
```diff
@@ -30,6 +30,12 @@ def write_version_py(filename=None):
     author='SimPhoNy FP7 European Project',
     description='The native implementation of the SimPhoNy cuds objects',
     long_description=README_TEXT,
+    install_requires=[
+        "cython",
```
Comment (dpinte) [inline setup.py]: is cython in install_requires or setup_requires?

**Comment by dpinte** [comment]
@itziakos except one comment, this LGTM

File: setup.py (hunk 1 above)
Comment (itziakos) [inline setup.py]: cood point. cython is actually only necessary for th depedencies not for building simphony-common it-self.

**Pull request merged**